static/scss/partials/main.scss

```diff
@@ -2367,12 +2368,20 @@ input.input-has-error {
   }
 }
 
+.messages {
+  width:100%;
+  padding:1.5rem 16px 0 16px;
+  margin: 0 auto;
+}
+
 .messages > div {
   background-color: var(--grey50);
 }
 
 .messages .success {
-  background-color: var(--blue7);
+  background-color: var(--successGreen);
+  border-radius:8px;
```

Comment:
Use the [Protocol spacing token](https://protocol.mozilla.org/fundamentals/tokens.html) here: `$spacing-sm`